shipyard_blueprints/mysql/shipyard_mysql/mysql.py

```diff
@@ -1,49 +1,227 @@
-import sys
-from sqlalchemy import create_engine, text
-from shipyard_templates import Database, ExitCodeException
-from sqlalchemy.engine import url
+import pandas as pd
+import os
+from sqlalchemy import create_engine
+from shipyard_templates import Database, ExitCodeException, ShipyardLogger
+from shipyard_templates.database import FetchError, UploadError, QueryError
+from sqlalchemy import create_engine, TextClause
+from shipyard_mysql.errors.exceptions import (
```

Comment:
If you opt to keep the exceptions this would be refactored once the errors folder is deleted